app/views/partials/personal_key/_key.html.erb

```diff
@@ -15,4 +12,33 @@
           date: content_tag(:strong, I18n.l(Time.zone.today, format: '%B %d, %Y')),
         ) %>
   </p>
+
+  <br />
```

Comment:
I'd suggest using margins to create spacing in the layout rather than a line break. I think you can remove this and change the `margin-y-0` on the `<p>` above to `<p class="margin-bottom-105">`.

Reply:
Changed.